guides/common/modules/proc_acd_installation.adoc

```diff
@@ -1,9 +1,8 @@
-[id="{context}_installation"]
-= Installing the ACD Plugins
+[id="{context}_installation_{ProjectServer}"]
+= Installing ACD on {ProjectServer}
 
-You need to install both the `foreman_acd` and `smart_proxy_acd` plugins to use ACD.
+ACD consists of two plugins: `foreman_acd` and `smart_proxy_acd`.
```

Comment:
Just a thought, but perhaps something like this?
```suggestion
ACD has a Foreman plugin (`foreman_acd`) and a Foreman Proxy plugin (`smart_proxy_acd`).
```

Reply:
Added!